src/test/groovy/graphql/GraphQLTest.groovy

```diff
@@ -139,7 +139,7 @@ class GraphQLTest extends Specification {
         then:
         errors.size() == 1
         errors[0].errorType == ErrorType.InvalidSyntax
-        errors[0].locations == [new SourceLocation(1, 8)]
+        errors[0].locations == [new SourceLocation(0, 8)]
     }
 
```

Comment:
` { hello }` is now considered as 0 based not 1's based

Reply:
Ahh I did not know the spec says that - I guess we need to add one,

Ones based index are weird right?? It goes against 50 years of computing theory! ;)